app/write_square_test.go

```diff
@@ -171,28 +168,25 @@ func Test_finalizeLayout(t *testing.T) {
 		},
 	}
 	for i, tt := range tests {
-		res, blobs := finalizeLayout(tt.squareSize, tt.nonreserveStart, tt.ptxs)
-		require.Equal(t, len(tt.expectedIndexes), len(res), i)
-		for j, ptx := range res {
-			assert.Equal(t, tt.expectedIndexes[j], ptx.shareIndexes, i)
-		}
+		t.Run(fmt.Sprintf("case%d", i), func(t *testing.T) {
```

Comment:
[super minor non-blocking question]
possibly stupid question, but what is the reasoning behind using t.Run here, but removing it elsewhere?

Reply:
I don't think I removed it anywhere. I don't think there's any major benefit but out of habit I've always broken these table tests into sub tests. In this case, adding the `fmt.Sprintf("case%d", i)` lets me work out faster what case failed.